src/app/data-model-provider/Provider.h

```diff
@@ -57,6 +59,15 @@ class Provider : public ProviderMetadataTree
     // event emitting, path marking and other operations
     virtual InteractionModelContext CurrentContext() const { return mContext; }
 
+    /// Validates that the given event path is supported, where path may contain wildcards.
+    ///
+    /// If any wild cards exist on the given path, the implementation is expected to validate
+    /// that a valid event path exists on some wildcard expansion.
+    ///
+    /// At the very minimum this will validate that a valid endpoint/cluster can be expanded
+    /// from the input path and that the given descriptor has access to it.
+    virtual bool EventPathReadable(const EventPathParams & path, const Access::SubjectDescriptor & descriptor) = 0;
```

Comment:
This needs a better name.  How about EventPathIncludesAccessibleConcretePath or something?